bin/thumbnail

```diff
@@ -80,6 +80,25 @@ cat <<EOF
 EOF
 }
 
+# Test for array containment.
+# Example: $(contains "${A[@]}", "foo") == "y"
+# Returns "y"" when the object is contained in the array and "n" otherwise.
+# http://stackoverflow.com/questions/3685970/check-if-an-array-contains-a-value
+function contains() {
+	local n=$#
+	local value=${!n}
+	for ((i=1;i < $#;i++)) {
+		if [ "${!i}" == "${value}" ]; then
+			echo "y"
```

Comment:
No, with `echo` you can capture the value in `$()`. If you `return` or `exit` then you have to check `$?`. Because `$()` is used below the output is captured.